includes/lib/transifex-live-integration-hreflang.php

```diff
@@ -107,10 +108,13 @@ private function generate_languages_hreflang( $raw_url, $languages,
 				$href_link_parts[0] = $protocol;
 				$arr['href'] = implode(':', $href_link_parts);
 			} else {
-				$arr['href'] = $url_map[$language];
+        $arr['href'] = $url_map[$language];
 			}
 			$arr['hreflang'] = $hreflang_map[$language];
+      // source language is placed first in the array
+      $arr['is_source'] = $firstIteration ? true : false;
```

Comment:
Maybe a tab can be added here so as this would be at the same level with line 116